packages/jest-did-matcher/src/matchers/toBeDidCoreDatetime/predicate.js

```diff
@@ -1,5 +1,5 @@
 export default expected => {
-    const regex = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z/
+    const regex = /[0-9]{4}-[0-1][0-9]-[0-3][0-9]T[0-2][0-9]:[0-6][0-9]:[0-6][0-9]Z/
```

Comment:
We don't observe 23:60:60 ?

```suggestion
    const regex = /[0-9]{4}-[0-1][0-9]-[0-3][0-9]T[0-2][0-9]:[0-5][0-9]:[0-5][0-9]Z/
```

Reply:
Of course, thanks :)